lib/util/extensions/num_extensions.dart

```diff
@@ -1,6 +1,17 @@
 import 'package:decimal/decimal.dart';
 import 'package:my_wit_wallet/constants.dart';
 import 'dart:math';
+import 'package:intl/intl.dart';
+
+final formatter = NumberFormat('#,###,000');
+String format(result) {
+  List parts = result.toString().split('.');
+  if (parts.length > 1) {
+    return '${parts[0].toString().length < 3 ? parts[0] : formatter.format(num.tryParse(parts[0]) ?? 0)}.${parts[1]}';
+  } else {
+    return '${parts[0].toString().length < 3 ? parts[0] : formatter.format(num.tryParse(parts[0]) ?? 0)}';
+  }
+}
```

Comment:
I think we could extract some repeated code. 
- We could add `parts[0].toString().length < 3` and `formatter.format(num.tryParse(parts[0]) ?? 0)}.${parts[1]}'` in variables.
- Return first `parts[0]` if the condition matches